INSTALL.md

```diff
@@ -0,0 +1,233 @@
+
+```sh
+meson setup build -Dprefix=/tmp/install
+```
+
+By default meson enables all Netatalk features if their dependencies are available.
+However, many features can be enabled or disabled at configure time.
+All such options take the form `-Dwith-option=true` or `-Dwith-option=false`:
+
+```
+Option                 Description                                 Default value
+
+with-acls              Enable ACL support                          true
+with-afpstats          Enable AFP statistics via D-Bus             true
+with-cracklib          Enable cracklib support                     true
+with-debug             Enable verbose debug code                   false
+with-dtrace            Enable DTrace probes                        true
+with-embedded-ssl      Enable built-in SSL support                 true
+                       (required for DHX/RANDNUM authentication)
+with-gsaapi            Enable gssapi support                       true
+                       (required for Kerberos V authentication)
+with-kerberos          Enable Kerberos V API                       true
+                       (optional for Kerberos V authentication)
+with-krb5-uam          Compile Kerberos V UAM                      true
+with-largefile         Enable largefile support                    true
+with-ldap              Enable LDAP support                         true
+with-libiconv          Enable iconv support                
+with-manual            Compile Netatalk html documentation         false
+with-overwrite         Overwrite existing configuration files      false
+with-pam               Enable PAM support                          true
+with-pgp-uam           Compile PGP UAM                             false
+with-quota             Enable Quota support                        true
+with-sendfile          Enable sendfile support                     true
+with-shadow            Enable shadow password support              true
+with-overwrite         Overwrite existing configuration files      false
+with-spotlight         Enable Spotlight support                    true
+with-tcp-wrappers      Enable TCP wrappers                         true
+with-tests             Compile tests                               false
+with-zeroconf          Enable Zeroconf (Bonjour) support           true
+```
+
+If meson cannot find the some required dependencies at configure time then the
+user can specify the path a library if it is installed in an unconventional
+location.
+All such options take the form -`Dwith-dependency-path=/path/to/dependency`,
+and the dependency directory must include lib and include directories.
+
+The following dependencies can be found in this way:
+
+```
+gssapi
+ldap
+libgcrypt
+docbook
+libiconv
+pam
+```
+
+Please see `meson_options.txt` for full details of all Netatalk-specific options,
+and https://mesonbuild.com/Builtin-options.html for details of generic Meson
+options.
+
+## External software dependencies
```

Comment:
On the topic of overdocumentation, the external dependencies are also [listed up in the manual](https://github.com/Netatalk/netatalk/blob/main/doc/manual/install.xml). The purpose is slightly different in that the manual explains in a bit more depth what each dependency does. Having both is probably a good idea.

Reply:
OK